CSM Vulnerabilities GovCloud Preview changes (#27177)
DataDog/documentation: content/en/security/cloud_security_management/vulnerabilities/_index.md

---

{{< site-region region="gov" >}}
<div class="alert alert-warning">Cloud Security Management Vulnerabilities is in Preview for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).

> Request access by <a href="https://www.datadoghq.com/product-preview/csm-vulnerability-management-in-govcloud">filling this form</a>.

Reviewer: ```suggestion
To request access, <a href="https://www.datadoghq.com/product-preview/csm-vulnerability-management-in-govcloud">complete the form</a>.
```
